homeassistant/components/samsungtv/config_flow.py

```diff
@@ -77,17 +75,60 @@ def _get_entry(self):
         }
         if self._bridge.token:
             data[CONF_TOKEN] = self._bridge.token
-        return self.async_create_entry(title=self._title, data=data,)
+        return self.async_create_entry(title=self._title, data=data)
+
+    def _abort_if_already_configured(self):
+        device_ip = gethostbyname(self._host)
+        for entry in self.hass.config_entries.async_entries(DOMAIN):
```

Comment:
We can call `self._async_current_entries` to get all existing entries for the integration.